Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Orbifold_Tutte_parameterizer_3.h

```diff
@@ -730,31 +716,30 @@ class Orbifold_Tutte_parameterizer_3
     // The other parts of M(i,j) and M(i,k) will be added when this function
     // is called from the neighboring faces of F_ijk that share the vertex i
 
-    // Compute: - tan(alpha / 2)
-    const NT w_i_base = 1.0 * compute_w_ij_mvc(pi, pj, pk);
-
     // @fixme unefficient: lengths are computed (and inversed!) twice per edge
 
+    // Set w_i_base: - tan(alpha / 2)
+    const Point_3& p = pk;
+    const Point_3& q = pi;
+    const Point_3& r = pj;
+    const CGAL::Weights::internal::
```

Comment:
Why these 3 aliases?

Reply:
Probably as later we have functions like `get_w_p()`.   Should pk,pi,pj then be changed?